tests/test-dirs/occurrences/project-wide/union.t

```diff
@@ -10,8 +10,8 @@ the marshal wouldn't be granular):
 
 A signature containing the same symbols:
 
-  $ echo "module type S = sig\n$(cat test.mli)\nend" >sig.ml
-  $ echo "module type S = sig\n$(cat test.mli)\nend" >sig.mli
+  $ echo "module type S = sig $(cat test.mli) end" >sig.ml
+  $ echo "module type S = sig $(cat test.mli) end" >sig.mli
   $ $OCAMLC -bin-annot -bin-annot-occurrences -c sig.mli sig.ml
 
 At this point `ŧest` and `sig` are unrelated. We'll later force their unification with:
```

Comment:
Off-topic but how did I even type the letter `t` with a strike-through in `ŧest`? o_o

Reply:
M̵̲̤̝̦̜͍̈́͊̑a̶̖̟̱͊̀y̶̛̛̹̹̯̬͕͆͑̀b̷̰̤͎͓͘ḙ̴̡̨̯̣̆̊́̏̇͘ ̵̭͔̌̃̀͛͌͝y̸̞̞̭̗͒͊̇o̶̧̥͓̕u̵͓̻͖̯̲͑̐̈̃͋̚͜r̶̮̀̍͒̕͝ ̴͍͖̮̥̤̣͆̾̄̿̊c̸̘̦̥̖̤͋̄̌o̷̹̼̎m̴͙̿̍͘͝ṕ̵̨̤̙͔̈̑́̐͘u̶̧̨͎̠͒̔̅t̸̬͒̈́͌̀̈́̅ě̴͓͛̇̃͠r̸͙͇̭͓͇̈̋͘ ̵̢̟̮̖̏͜i̶̮͉͙̅̀̄̈́̈́͝s̶͖͕̒̒͝ ̴̤̻͐ċ̸̠̫̟̩͔͑̈̊͐͑u̸̘͔̠͊͂͒̽͂r̵̠̾̽̈́̓̒ś̵̹̻͍ĕ̴̦̘̯̬͘͝d̶̞̩̂͛̄̕ ̷̡̢̣̌?̸̮̐